catalog/spatial/geocoding/spatial-clavin-lucene/src/main/java/org/codice/ddf/spatial/clavin/ClavinFacade.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+
+package org.codice.ddf.spatial.clavin;
+
+import java.util.List;
+
+import com.bericotech.clavin.resolver.ResolvedLocation;
+
+/**
+ * A {@code ClavinFacade} provides access to the clavin lucene index.
+ */
+public interface ClavinFacade {
+    /**
+     * Get a list of clavin ResolvedLocation.
+     *
+     * @throws ClavinFacadeException if an error occurs while accessing the index.
+     */
+    List<ResolvedLocation> getResolvedLocations(String document) throws ClavinFacadeException;
```

Comment:
Is "document" a required parameter? Please document what the parameter means, and what happens if null is passed in.